src/zos_jobs/zowe/zos_jobs_for_zowe_sdk/jobs.py

```diff
@@ -65,6 +65,8 @@ def cancel_job(self, jobname, jobid):
             The name of the job
         jobid: str
             The job id on JES
+        modify_version: str
+            Default ("2.0") specifies that the request is to be processed synchronously. For asynchronous processing - change the value to "1.0"
 
```

Comment:
**IMHO**, it would be better to use a meaningful name, like "async_req=True", instead of "modify_version=..".
Also, as far as i can see https://www.ibm.com/docs/en/zos/2.4.0?topic=interface-cancel-job-purge-its-output, "default" value is "async", and sync works only for JES2. 
It is just my opinion. Thank you.

Reply:
Alright, thank you, might make some edits here as well! Read the docs too, but I was just following the request from the opened issue:
"... we should add an optional modifyVersion parameter to the cancel_job API with a default value of "2.0" (synchronous). Users can set it to "1.0" if they prefer asynchronous processing."

Reply:
I would recommend to stick to the description of the issue and interpret it. There was probably some thought behind the decision to request a `modify_version` parameter over a different one. If you think the design is not right, this should be discussed on a call with the squad and if a different design is considered, the issue should be updated or a new one should be created.